src/styles/c-global-style.scss

```diff
@@ -90,19 +90,34 @@ $interaction-types3: primary, dark;
     }
   }
 }
-.bg {
-  @each $type in join(join($interaction-types, $interaction-types2), $interaction-types3) {
-    &-#{$type} {
-      background-color: map-get($interaction-colors, $type) !important;
-      background-color: var(--#{$type}) !important;
+
+$util-elems : bg, text, border;
+
+@each $elem in $util-elems {
+  .#{$elem} {
+    @if($elem == bg) {$elem : background;}

```

Comment:
Maybe create a map containing the real property instead of  having conditions here?

Reply:
Added improvement update https://github.com/scania/scania-theme/pull/9/commits/ecc2b53c48839459b40fb861875abee88f9c9d07